examples/fault-injection/README.md

```diff
@@ -0,0 +1,44 @@
+Envoy fault injection example
+=============================
+
+In this example, we show [Envoy's fault injection
+feature](https://www.envoyproxy.io/docs/envoy/latest/configuration/http_filters/fault_filter) in a sandbox.
+
+## Usage
+```
+# in terminal 1
+docker-compose up
+
+# in terminal 2
+docker-compose exec envoy bash
+bash send_request.sh
+```
+
+This constantly sends HTTP GET requests to running Envoy process. The process proxies the requests to the backend server
+which is running python process in the "backend" container. We see HTTP 200 responses here.
+
+Then let's inject a fault using [Envoy's runtime
+feature](https://www.envoyproxy.io/docs/envoy/latest/configuration/runtime).
+
+```
+# in terminal 3
+docker-compose exec envoy bash
+bash enable_fault_injection.sh
+```
+
+This changes Envoy's fault injection filter config via runtime setting. Then `send_request.sh` script shows HTTP 503
```

Comment:
The script above enables HTTP aborts for 100% of requests. So, you should now see a continuous sequence of HTTP 503 responses for all requests.